packages/core/phase2/lib/getOperationSequence/areAllResultsAlreadyPresentOnPnc/isMatchToPncAdjAndDis/isMatchToPncDis.ts

```diff
@@ -1,17 +1,19 @@
-import type { AnnotatedHearingOutcome } from "../../../../../types/AnnotatedHearingOutcome"
+import type { Result } from "../../../../../types/AnnotatedHearingOutcome"
 import type { ExceptionResult } from "../../../../../types/Exception"
 import type { PncDisposal } from "../../../../../types/PncQueryResult"
-import { getDisFromResult } from "./getDisFromResult"
-import isPncDisposalMatch from "./isPncDisposalMatch"
+import areDisposalsMatch from "./areDisposalsMatch"
+import { createDisposalByResult } from "./createDisposalByResult"
 
 const isMatchToPncDis = (
```

Comment:
Could we name this to `isMatchToPncDisposal`?

Reply:
Renamed. ✔️